content/_data/changelogs/weekly.yml

```diff
@@ -336,7 +336,69 @@
         When validating a cron expression, consider the specified time zone.
       pull: 2824
       issue: 43228
-
+- version: "2.55"
+  date: 2017-04-15
+  changes:
+    - type: major bug
+      message: >
+        Packaging:
+        Debian package now requires Java 8.
+        (causes regression since 2.54).
+      issue: 43495
+    - type: rfe
+      message: >
+        Recommended plugins:
+        Replace the deprecated <a href="https://plugins.jenkins.io/github-organization-folder">
+        GitHub Organization Folder</a> plugin
+        by <a href="https://plugins.jenkins.io/github-branch-source">
+        GitHub Branch Source</a>
+        and <a href="https://plugins.jenkins.io/pipeline-github-lib">Pipeline: GitHub Groovy Libraries</a> plugins.
+      pull: 2842
+    - type: rfe
+      message: >
+        Added fine-grain logging of <code>FullDuplexHttpService</code> to diagnose issues when establishing an HTTP Duplex connection.
+      pull: 2481
+    - type: bug
+      message: >
+        Update LibZFS from 0.5 to 0.8 to fix compatibility issues with ZFS filesystem and illumos distributions.
+      issue: 41932
+    - type: bug
+      message: >
+        Before deleting jobs, try to abort the pending builds.
+        If abortion fails, an error will be thrown instead of the job deketion.
+      issue: 35160
+    - type: bug
+      message: >
+        Ensure that <code>Cloud.PROVISION</code> is properly initialized during the configuration loading.
+      issue: 37616
+    - type: bug
+      message: >
+        Fix log message formatting when migrating `AllView` names
+        due to <a href="https://issues.jenkins-ci.org/browse/JENKINS-38606">JENKINS-38606"</a>.
+      issue: 43611
+    - type: bug
+      message: >
+        Setup wizard gets into bad state when failures like network issues happen.
+      issue: 41778
+    - type: bug
+      message: >
+        Catch and log <code>RuntimeException</code> in <code>Computer#setNode()</code> when updating the Computer list.
+      issue: 42043
+    - type: bug
+      message: >
+        SSH CLI client authenticator 1.4. Add missing SSH Public Key field validation in user configuration.
+      issue: 16337
+    - type: rfe
+      message: >
+        Internal API: 
+        SSH CLI client authenticator 1.3. 
+        Expose <code>PublicKeySignatureWriter</code> to plugins.
+      pull: 2480
+    - type: rfe
+      message: > 
+        Build Flow:
```

Comment:
Ambiguous. If this needs to be mentioned (I disagree, nobody cares), should be "Internal".